examples/gitlab-asg/Makefile

```diff
@@ -58,45 +91,46 @@ plan:
 apply:
 	@terraform apply tf.out
 
-## Generate the ssh_congig both for Git operations and instance maintenance
+## Generate the ssh_config both for Git operations and instance maintenance
 render-ssh-config:
-	@. ./vars.env && envsubst <ssh_config.tpl > ssh_config
-	@echo `aws ec2 describe-instances --filters "Name=tag:Name,Values=at-test-gitlab-server-eu-central-1a" --region=eu-central-1 | jl _.Reservations[0].Instances[0].PublicIpAddress | tr -d '"'` >> ssh_config
+	@export SERVER_NAME=$$(terraform output gitlab_server_name) && \
+	 export SERVER_IP=$$(./server-ip.sh $$SERVER_NAME $$REGION) && \
+	 envsubst <ssh_config.tpl >ssh_config
 
 ## use ops cli tool to lookup the public IP of the gitlab instance
 ##get-ip:
 ##	@ops aws ec2 asg ips -r $$(terraform output region) $$(terraform output gitlab_asg_name)
 
 ## SSH to the gitlab server and use parted and similar tools to setup a partition and filesystem on the EBS volume (run only once per volume)
 init-ebs-volume:
-	@ssh data-ops-gitlab "sudo parted --script /dev/xvdf -- mklabel msdos && \
+	@ssh -F ssh_config data-ops-gitlab "sudo parted --script /dev/xvdf -- mklabel msdos && \
```

Comment:
is the `data-ops-gitlab` bit a hardcoded relic from my previous work? Does that need to be parametized somewhere?

Reply:
That name is now purely local, because of using a custom `ssh_config` instead of putting the entries into `~/.ssh/config`. So the name is arbitrary, and it actually makes things easier if it's always the same.